src/js/binaryen.js-post.js

```diff
@@ -4340,34 +4238,34 @@ Module['TupleMake'] = makeExpressionWrapper({
       Module['_BinaryenTupleMakeRemoveOperandAt'](expr, --prevNumOperands);
     }
   },
-  'getOperandAt': function(expr, index) {
+  'getOperandAt'(expr, index) {
```

Comment:
I wasn't aware this type of syntax is possible (with quotes). Is it common?

Reply:
It introduced with ES6 / ES2015 so it's not widely known